src/qt/clientmodel.h

```diff
@@ -108,7 +110,8 @@ class ClientModel : public QObject
     void showProgress(const QString &title, int nProgress);
 
 public Q_SLOTS:
-    void updateTimer();
+    void updateTimer1();
+    void updateTimer2();
```

Comment:
For readability, I think it would make more sense to name these functions `updateTimer_TPS` and `updateTimer_Ban`, since the original deals with updating the transactions per second and the new one deals with clearing expired bans.